.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+README1.md
```

Comment:
- [ OPTIONAL] The readme should not be ignored. This file guides other people on how to use your project. It should therefore be pushed to the repo. 👍🏾